lib/mayaUsd/nodes/proxyAccessor.py

```diff
@@ -345,11 +347,11 @@ def __parent(doParenting, forceUnparenting=False):
    
    parentItems(ufeChildren, ufeParent, doParenting)
 
-def parent(force=False):
-    __parent(True, forceUnparenting=force)
+def parent(*ufeItemPathStrings, force=False):
```

Comment:
This change is to provide more flexibility? I don't see anything calling this in this PR

Reply:
`ProxyAccessorUndoItem::parentPulledObject` now passes the child and parent ufe paths as `proxyAccessor,parent` arguments. This is to avoid modifying maya's selection 

https://github.com/jufrantz/maya-usd/commit/008b0da#diff-f40afd2ca9921208caf4e38e91fa4ab00a2aa102b6fcf54fc82fdb4ff26427fdR46